package.json

```
@@ -9,6 +9,7 @@
     "axios": "^0.21.1",
     "bootstrap": "3.4.1",
     "country-list": "2.2.0",
+    "dompurify": "^2.2.9",
```

nit: I prefer not to use the caret. In the past, I've hit very difficult-to-debug problems from this. NPM library maintainers have an unfortunate history of introducing bugs in minor patches and this could pull in a version we haven't tested against ([link](https://michaelsoolee.com/npm-package-tilde-caret/)):
> If you see ^1.0.2 it means to install version 1.0.2 or the latest minor or patch version such as 1.1.0

For that reason, I like to pull in exactly what we have tested against:
```suggestion
    "dompurify": "2.2.9",
```